lib/Conversion/TritonLangToTriton/TritonLangToTritonPass.cpp

```diff
@@ -0,0 +1,604 @@
+#include "triton/Conversion/TritonLangToTriton/TritonLangToTritonPass.h"
+#include "mlir/Conversion/LLVMCommon/TypeConverter.h"
+#include "mlir/Dialect/Arith/IR/Arith.h"
+#include "mlir/Dialect/ControlFlow/IR/ControlFlow.h"
+#include "mlir/Dialect/ControlFlow/IR/ControlFlowOps.h"
+#include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/Math/IR/Math.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/Dialect.h"
+#include "mlir/IR/FunctionInterfaces.h"
+#include "mlir/Interfaces/CallInterfaces.h"
+#include "mlir/Interfaces/ControlFlowInterfaces.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/DialectConversion.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+#include "triton/Analysis/Utility.h"
+#include "triton/Dialect/Triton/IR/Dialect.h"
+#include "triton/Dialect/TritonLang/IR/Dialect.h"
+#include "llvm/ADT/APSInt.h"
+#include <functional>
+#include <numeric>
+#include <queue>
+
```

Comment:
Currently, this pass only does the following thing:

1. Replacing the constexpr arguments with the actual constant values
2. Do shape inference
3. Eagarly replace the TL ops with TT ops